Automattic/wp-calypso
# Jetpack Connect: Remove unnecessary plan features tab stylesheet

It seems that when we removed the plan tabs test in #11525, we forgot to remove the [unnecessary stylesheet](https://github.com/Automattic/wp-calypso/blob/f6af7ddc3032122aa3cf72daf7ecaaf7aadeed98/client/jetpack-connect/plan-features-tab/style.scss). This PR removes it.

To test:
* Checkout this branch
* Verify the stylesheet is not used anywhere.
